### ISSUE TITLE
Short line lengths ... why?

### ISSUE THREAD

**Issue by nellyt** [gjr80/weewx-polarwindplot#26]
https://github.com/gjr80/weewx-polarwindplot/blob/fc8738b446e0d9cd447a7d044309b1b942cbd056/bin/user/polarWindPlot.py#L1527

In some places you seem to impose an 80 column limit on lines, in others you go a bit over

Is that how your editor is configured or do you see this as a RULE ?

80 columns used to be in most coding standards but that rule died a death when monitors got bigger!

**Comment by gjr80** [comment]
It largely stem from [PEP 8](https://www.python.org/dev/peps/pep-0008/), the Python style guide. Have a read through, there are some useful principles in there.

In terms of line length I try to limit my code to 78 or 79 characters per line though I will go a few characters over if breaking the line leaves only a couple of characters on the following line. If a line cannot be easily broken I will leave it long, eg a long equation may need numerous brackets to allow it to be split over multiple lines, adding these brackets may make it hard to read so I tolerate the long line. 

Bottom line I guess is it's not a hard and fast rule but I tend to follow a few principles. And my editor (Notepad++) has a nice blue vertical line at 79 characters as a guide :)
